include/net/nrf_cloud_cell_pos.h

```diff
@@ -36,6 +36,20 @@ struct nrf_cloud_cell_pos_result {
 	uint32_t unc;
 };
 
+/** @defgroup nrf_cloud_cell_pos_omit Omit item from cellular positioning request.
+ * @{
+ */
+#define NRF_CLOUD_CELL_POS_OMIT_TIME_ADV	LTE_LC_CELL_TIMING_ADVANCE_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_RSRQ		LTE_LC_CELL_RSRQ_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_RSRP		LTE_LC_CELL_RSRP_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_EARFCN		(LTE_LC_CELL_EARFNC_MAX + 1)
+/** @} */
+
+/* TODO: Confirm this value...
+ * Is it actually different than LTE_LC_CELL_TIMING_ADVANCE_MAX ?
+ */
+#define NRF_CLOUD_CELL_POS_TIME_ADV_MAX		7690
```

Comment:
I don't know where this value comes from, but in the modem AT specification the maximum value for TA is 20512.

Reply:
this is the max value specified by the API: https://api.nrfcloud.com/v1#operation/GetPositionFromCellTowers
i posted in the positioning channel to get more info on this parameter

Reply:
nrf cloud will update the API... max will be 20512